### PR TITLE
new CrayGNU files for 2015.06-XC 

### PR DESCRIPTION
**Pull request by gppezzi**
This replaces PR #2432

### CODE REVIEW

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/ff4080c7ba54532073c2 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 25 out of 25 (22 easyconfigs in this PR)
Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/e9431f57c562cfa1fcf9 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/a89afa162e72da5aa7da for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 30 out of 30 (22 easyconfigs in this PR)
brisi101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/cb1f82c861a996dcfe34 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
dora105 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/85043952753ac5b4cbce for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/8754bcad2fe8df8c2522 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
daint105 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.6
See https://gist.github.com/e2a5809ffd33a4826107 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 30 out of 30 (22 easyconfigs in this PR)
santis101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.6
See https://gist.github.com/d4e040ed99c7271d6cb9 for a full test report.

**Comment by gppezzi** [comment]
Tests passed on Daint, Dora, Santis, Brisi and Swan, from my side everything looks good to merge this.

@boegel @pforai Could you please try this PR on sisu?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 20 out of 22 (22 easyconfigs in this PR)
sisu-login1 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/87ed25b879310188668e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
sisu-login1 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/fb94e12574ec04ad4582 for a full test report.

**Comment by boegel** [comment]
Problem with GROMACS on Sisu is caused by incomplete metadata file for external modules, more specifically because there is no metadata for the external `fftw` module being used; retrying with metadata file from https://github.com/hpcugent/easybuild-framework/pull/1586.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.7-CrayGNU-2015.06-XC-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+source_urls = [
+    'ftp://ftp.gromacs.org/pub/gromacs/',  # GROMACS sources
+    'http://gerrit.gromacs.org/download/',  # regression tests sources
+]
+sources = [
+    SOURCELOWER_TAR_GZ,
+    'regressiontests-%(version)s.tar.gz',
+]
+
+preconfigopts  = "export CMAKE_LIBRARY_PATH=$CMAKE_LIBRARY_PATH:${EBROOTFFTW} && "
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.7-CrayGNU-2015.06-XC-mpi.eb]: this is not 100% correct, see also https://github.com/hpcugent/easybuild-easyconfigs/pull/2434/files#r54280136 (let's keep the discussion there)

Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.7-CrayGNU-2015.06-XC-mpi.eb]: fixed in https://github.com/gppezzi/easybuild-easyconfigs/pull/1

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 22 out of 23 (22 easyconfigs in this PR)
sisu-login2 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/3d97b026e9e6519e9932 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
dora101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/725ad6d374eca3cf2894 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 30 out of 30 (22 easyconfigs in this PR)
brisi101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/59869a2fb0e6e34e63c4 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
swan - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/d1dd9f723c28131dc6b0 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**FAILED**
Build succeeded for 21 out of 22 (22 easyconfigs in this PR)
daint101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.6
See https://gist.github.com/e6b68163286497179d4a for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 30 out of 30 (22 easyconfigs in this PR)
santis101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.6
See https://gist.github.com/be0ef999e151204ca943 for a full test report.

**Comment by boegel** [comment]
@gppezzi: any idea why scipy failed on Daint?

**Comment by boegel** [comment]
```
ImportError: /dev/shm/perettig/easybuild/software/numpy/1.9.2-CrayGNU-2015.06-XC-Python-2.7.9/lib/python2.7/site-packages/numpy/core/multiarray.so: undefined symbol: PyUnicodeUCS4_AsUnicodeEscapeString
```

This *may* be related to https://github.com/hpcugent/easybuild-easyblocks/pull/817

**Comment by boegel** [comment]
the scipy failure is probably caused by a bug in framework (robot not correctly ordering builds), cfr. https://github.com/hpcugent/easybuild-framework/issues/1640

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 31 out of 31 (22 easyconfigs in this PR)
daint104 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.6
See https://gist.github.com/c085db1cbc36496c43e0 for a full test report.

**Comment by boegel** [comment]
Going in, just like #2434, thanks for your efforts @gppezzi!

**Pull request merged**